### DISCUSSION TITLE
Higher HRNet multi-scale results on CrowdPose question

### DISCUSSION THREAD

**Discussion by nibox** [open-mmlab/mmpose#489] [Q&A]
Hi, 
first of all, thank you very much for reproducing the results on CrowdPose and publishing your model.
My question is do you have results for HigherHRNet w32 on the CrowdPose test split using multi-scale inference?
I tried to produce them by setting test_scale_factor=[2.0, 1.0, 0.5] in the val_pipline config, but the results are worse compared to single-scale inference, so I am not sure if I have done everything correct.

**Comment by jin-s13** [comment]
Yes, you just need to set `test_scale_factor` in the val_pipline config.
I have tried the `test_scale_factor=[2, 1]`, and the results look good.


 Average Precision  (AP) @[ IoU=0.50:0.95 | type=   all | maxDets= 20 ] = 0.661
 Average Precision  (AP) @[ IoU=0.50      | type=   all | maxDets= 20 ] = 0.864
 Average Precision  (AP) @[ IoU=0.75      | type=   all | maxDets= 20 ] = 0.710
 Average Recall     (AR) @[ IoU=0.50:0.95 | type=   all | maxDets= 20 ] = 0.734
 Average Recall     (AR) @[ IoU=0.50      | type=   all | maxDets= 20 ] = 0.912
 Average Recall     (AR) @[ IoU=0.75      | type=   all | maxDets= 20 ] = 0.778
 Average Precision  (AP) @[ IoU=0.50:0.95 | type=  easy | maxDets= 20 ] = 0.742
 Average Precision  (AP) @[ IoU=0.50:0.95 | type=medium | maxDets= 20 ] = 0.670
 Average Precision  (AP) @[ IoU=0.50:0.95 | type=  hard | maxDets= 20 ] = 0.566



I will try `test_scale_factor=[2.0, 1.0, 0.5]` today. Will let you know when I get the results.

**Comment by jin-s13** [comment]
I have just tried `test_scale_factor=[2.0, 1.0, 0.5]`. The results are listed below.

Average Precision  (AP) @[ IoU=0.50:0.95 | type=   all | maxDets= 20 ] = 0.614
Average Precision  (AP) @[ IoU=0.50      | type=   all | maxDets= 20 ] = 0.801
Average Precision  (AP) @[ IoU=0.75      | type=   all | maxDets= 20 ] = 0.656
Average Recall     (AR) @[ IoU=0.50:0.95 | type=   all | maxDets= 20 ] = 0.710
Average Recall     (AR) @[ IoU=0.50      | type=   all | maxDets= 20 ] = 0.895
Average Recall     (AR) @[ IoU=0.75      | type=   all | maxDets= 20 ] = 0.748
Average Precision  (AP) @[ IoU=0.50:0.95 | type=  easy | maxDets= 20 ] = 0.699
Average Precision  (AP) @[ IoU=0.50:0.95 | type=medium | maxDets= 20 ] = 0.627
Average Precision  (AP) @[ IoU=0.50:0.95 | type=  hard | maxDets= 20 ] = 0.501

It seems that including 0.5 in the test scales is not always beneficial.

**Comment by nibox** [comment]
Thank you very much. For test_scale_factor=[2.0, 1.0, 0.5] i get the same results.
Now I am wondering which configuration of scales factors the original authors used.
